dev/MRTCore/SignConfig.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<SignConfigXML>
```

Comment:
Do these signing files need updating?  (Paging @EHO-makai )

Reply:
We only have test signing up and running right now, if this sub-module has flight (or similar release) signing working, we should leave that in place until we figure out the short-term posture.  I'll email Hui and Jevan today and we will decide.  I think this can go in as-is.

Reply:
Looks like PackageES signing is turned on based on the yml pipeline file I reviewed. We should leave that in place